packages/app/src/containers/ExportInspector.tsx

```diff
@@ -17,15 +17,20 @@ import {
 } from '../contexts/ApplicationStateContext';
 import useCanvasKit from '../hooks/useCanvasKit';
 import { renderImageFromCanvas } from '../utils/renderImageFromCanvas';
+import JSZip from 'jszip';
 
-async function saveFile(name: string, data: Uint8Array) {
+async function saveFile(
+  name: string,
+  fileFormat: string,
+  data: Uint8Array | ArrayBuffer,
```

Comment:
The type here can be just `ArrayBuffer`. If we need to get an `ArrayBuffer` from a `Uint8Array`, we can use `uint8Array.buffer`.